### ISSUE TITLE
Improve studio dev server workflow

### ISSUE THREAD

**Issue by schickling** [prisma/studio#177]
Instead of having a global `studio-server` CLI, I propose the following set up:

- Get rid of `studio-server` CLI
- Get rid of the `studio-server.prefs` file
- Add new `dev` script to top-level `package.json` in `studio-code` which takes the following CLI args `--schema /my/project/path/prisma/schema.prisma` and an optional `--port 12345` (default: `55555`) arg.
  - The command automatically starts the react dev server + listens to the schema changes in the provided path
  - This should also call `open(...)` to open the dev server directly in the browser

**Comment by idan** [comment]
Let's discuss at sync with context of what studio is turning into with prisma dev/init/etc

**Comment by janpio** [comment]
@madebysid This is documented in https://github.com/prisma/studio-code#setup, so the mentioned `dev` script was turned into just `npm start`?

**Comment by madebysid** [comment]
@janpio Yep. All packages use this naming schema, so seemed appropriate.

**Comment by janpio** [comment]
Yep, just wanted to persist that change here for future reference when reading this issue.
